### ISSUE TITLE
getting low accuracy in Avocado

### ISSUE THREAD

**Issue by divya9118** [cogsys-tuebingen/deephs_fruit#13]
I followed the same steps to train model, didnt made any changes. But I am getting 0.9 to 1 training accuracy but test accuracy is about 0.25 for Avocado.  Can you explain why is it so. And how can I solve this problem.

**Comment by leonvarga** [comment]
Low test accuracy means your model is overfitting. Please check the training procedure and monitor the training / validation loss. Depending on the used library version, the training procedure can differ. Also, a different batch size or the  usage of multiple GPUs require adaptations of the training.

**Comment by divya9118** [comment]
see for all other categories, accuracy is approx same as given in research paper. The problem persist only while training Avacado for VIS camera type(in both ripeness and firmness category). Even SVM and KNN are also having low accuracy for Avacodo (VIS type). It seems like there is some issue with VIS data for Avocado.

**Comment by leonvarga** [comment]
Hi,

sorry, without your whole setup, it is really hard to check where this error is coming from.

The avocado recordings were used for a couple of other works, like [https://github.com/cogsys-tuebingen/hsi_benchmark](https://github.com/cogsys-tuebingen/hsi_benchmark), and we did not experience any errors. 

You can check which samples are not correctly classified and try to check whether you see some pattern. Still, it seems your training is overfitting. 

Especially, the avocado VIS training should be really stable.
